src/open_inwoner/cms/banner/models.py

```diff
@@ -5,8 +5,12 @@
 from filer.fields.image import FilerImageField
 
 
-class Banner(CMSPlugin):
-    title = models.CharField(_("title"), max_length=250, default="")
+class BannerImage(CMSPlugin):
+    name = models.CharField(
+        _("Name"),
+        max_length=250,
+        help_text=_("The name of the image (this will not be shown on the page)"),
+    )
```

Comment:
Why do we need a name-field if we're not showing it?

Reply:
I was thinking that we do not have any required fields, so maybe I should remove name and make image a required field?

Reply:
Discussed with Bart, name fields are not useful. Image and title will be the required fields for the two models (`BannerImage` and `BannerText`).

Reply:
Reasoning why I didn't request to remove 'name' is that I often see plugins with an (internal) name which is only used for plugin.__str__ , this to make the name visible in the CMS when editting the page.

This is mostly used for CMS pages with long lists of plugins, so I agree that the benefit for only the bannerplugin is quite limited.